src/SprykerEco/Zed/Easycredit/Communication/Plugin/Oms/Condition/IsOrderConfirmedPlugin.php

```diff
@@ -28,6 +33,6 @@ class IsOrderConfirmedPlugin extends AbstractPlugin implements ConditionInterfac
      */
     public function check(SpySalesOrderItem $orderItem)
     {
-        return $this->getFacade()->sendOrderConfirmationRequest($orderItem->getFkSalesOrder())->getConfirmed();
+        return $this->getFacade()->sendOrderConfirmationRequest($orderItem->getFkSalesOrder())->getConfirmedOrFail();
```

Comment:
Is it safe to call `getConfirmedOrFail`? Maybe it makes sense to cast it to bool?